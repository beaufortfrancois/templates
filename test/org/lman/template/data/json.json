{
  "deep": [
    {
      "one": {
        "foo": "a",
        "bar": "b"
      }
    },
    {
      "two": {
      "foo": "c",
      "bar": "d"
      }
    }
  ]
}
