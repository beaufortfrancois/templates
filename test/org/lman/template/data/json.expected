hello
[{"one":{"foo":"a","bar":"b"}},{"two":{"foo":"c","bar":"d"}}]
world
